File: 코딩 기초 트레이닝/홀수 vs 짝수-p181887/홀수 vs 짝수.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181887

function solution(num_list) {
    var answer = [0,0];
    // answer[0] 홀수
    // answer[1] 짝수
    num_list.forEach((n, index) => {
        if((index + 1) % 2 === 0) {
            answer[1] += n;
        } else {
            answer[0] += n;
        }
    });

    return answer[0] >= answer[1] ? answer[0] : answer[1];
}

File: 코딩 기초 트레이닝/홀수 vs 짝수-p181887/홀수 vs 짝수.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181887

def solution(num_list):
    answer = [0,0]

    for index, num in enumerate(num_list):
        if (index+1) % 2 == 0:
            answer[0] += num
        else:
            answer[1] += num

    return answer[0] if answer[0] >= answer[1] else answer[1]
